### PR TITLE
Rename dppl to dpctl

### PR DESCRIPTION
**Pull request by 1e-to**
Closes the https://github.com/IntelPython/dpctl/issues/31

### CODE REVIEW

File: dpctl-capi/include/dpctl_sycl_program_interface.h [hunk 1]
```diff
@@ -85,10 +85,10 @@ DPPLProgram_CreateFromOCLSource (__dppl_keep const DPPLSyclContextRef Ctx,
...
  * @return   A SyclKernel reference if the kernel exists, else NULL
  */
-DPPL_API
-__dppl_give DPPLSyclKernelRef
-DPPLProgram_GetKernel (__dppl_keep DPPLSyclProgramRef PRef,
-                       __dppl_keep const char *KernelName);
+DPCTL_API
+__dpctl_give DPCTLSyclKernelRef
+DPCTLProgram_GetKernel (__dpctl_keep DPCTLSyclProgramRef PRef,
+                       __dpctl_keep const char *KernelName);
```
Comment (diptorupd) [inline dpctl-capi/include/dpctl_sycl_program_interface.h:91]: leading white space.

File: dpctl-capi/include/dpctl_sycl_program_interface.h [hunk 2]
```diff
@@ -56,10 +56,10 @@ DPPL_C_EXTERN_C_BEGIN
...
  *           else returns NULL.
  */
-DPPL_API
-__dppl_give DPPLSyclProgramRef
-DPPLProgram_CreateFromOCLSpirv (__dppl_keep const DPPLSyclContextRef Ctx,
-                                __dppl_keep const void *IL,
+DPCTL_API
+__dpctl_give DPCTLSyclProgramRef
+DPCTLProgram_CreateFromOCLSpirv (__dpctl_keep const DPCTLSyclContextRef Ctx,
+                                __dpctl_keep const void *IL,
```
Comment (diptorupd) [inline dpctl-capi/include/dpctl_sycl_program_interface.h:62]: leading white spaces. The issue exists in several places, may be it is a tabsize issue please take a look.

**Review by diptorupd — APPROVED** [review]
Some minor formatting nitpicks, but otherwise ready to go. Thanks for taking care of it.

File: dpctl-capi/include/dpctl_sycl_program_interface.h (hunk 1 above)
Comment (PokhodenkoSA) [reply dpctl-capi/include/dpctl_sycl_program_interface.h]: Unfortunately we have no C/C++ formatter yet.

**Comment by PokhodenkoSA** [comment]
I am waiting for a new "Great job!" from daal4py again!
Release notes should highlight this changes twice.

**Pull request merged**